examples/chunking/app.ddlog

```diff
@@ -18,13 +24,16 @@ word_features(
 
 tag?(word_id bigint) Categorical(13).
 
-function ext_training
-  over rows like words_raw
-  returns rows like words
-  implementation "udf/ext_training.py" handles tsv lines.
-
-words +=
-  ext_training(wid, word, pos, tag) :- words_raw(wid, word, pos, tag).
+words(sent_id, word_id, word, pos, true_tag, tag_id) :-
+  words_raw(sent_id, word_id, word, pos, true_tag),
+  tags(tag_id, tag),
+  if true_tag = "B-UCP" then ""
+  else if true_tag = "I-UCP" then ""
```

Comment:
With these two exceptions, did the result become identical to the old example's?